main.go

```diff
@@ -484,8 +486,38 @@ func (carina *ClusterCommand) Get(pc *kingpin.ParseContext) (err error) {
 }
 
 // Delete a cluster
-func (carina *ClusterCommand) Delete(pc *kingpin.ParseContext) (err error) {
-	return carina.clusterApply(carina.ClusterClient.Delete)
+func (carina *CredentialsCommand) Delete(pc *kingpin.ParseContext) (err error) {
+	err = carina.clusterApply(carina.ClusterClient.Delete)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "Unable to delete cluster, not deleting credentials on disk")
+		return err
+	}
+	p, err := carina.clusterPath()
+
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to locate carina config path, not deleteing credentials on disk\n")
+		return err
+	}
+
+	// TODO: Any other failsafes we should check here?
+	if p == "" || p == "/" {
```

Comment:
Oh I see you are using path everywhere, and never filepath. So path won't ever be in Windows format.

When you pass in a path created by `path.Join` into `ioutil.WriteFile`, does the package handle figuring out the proper Windows path?